packages/components/src/ecosystem_bar/index.jsx

```diff
@@ -0,0 +1,43 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { useRef } from 'react';
+import { useOnceSwipeTracking, useScrolled } from '@/src/ecosystem_bar/hooks';
+import styles from './index.module.scss';
+
+const EcosystemBar = ({ items, onFirstSwipe, onItemClick }) => {
+  const navRef = useRef(null);
+  const scrolled = useScrolled(navRef);
+
+  useOnceSwipeTracking(navRef, onFirstSwipe);
+
+  return (
+    <ul className={clsx(styles.root, { [styles.scrolled]: scrolled })} ref={navRef}>
+      {items.map((item) => (
+        <li key={item.key} className={styles.item}>
+          <a
```

Comment:
Maybe update PopupLink component and use it in here?

Reply:
Not sure about that one since nebenan-components is deprecated, I think in this case it's okayish to have it like that. Maybe we can migrate PopupLink to goodhood later on, if find more use cases like these?